apps/lumberjack-app/project.json

```
@@ -84,6 +84,7 @@
     },
     "use-lumberjack-bundle": {
       "executor": "nx:run-commands",
+      "dependsOn": ["^build"],
```

suggestion: Use explicit configuration to make it clearer.
```suggestion
      "dependsOn": [{ "projects": "dependencies", target: "build" }],
```